assets/css/objects/_form.scss

```diff
@@ -0,0 +1,15 @@
+.form {
+  h2 {
```

Comment:
This is a smell to me. Why all `h2`s in `form`s? Can we class this and make it more reusable?